runtime/rasdump/javadump.cpp

```diff
@@ -5760,52 +5778,76 @@ regionIteratorCallback(J9JavaVM* virtualMachine, J9MM_IterateRegionDescriptor* r
 }
 
 #if JAVA_SPEC_VERSION >= 21
+static jvmtiIterationControl
+vthreadCountIteratorCallback(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData) {
+	U_32 *vthreadCounts = (U_32 *)userData;
+	j9object_t vthread = J9VMJDKINTERNALVMCONTINUATION_VTHREAD(vmThread, object->object);
+	UDATA vThreadState = J9VMJAVALANGVIRTUALTHREAD_STATE(vmThread, vthread);
+#ifndef JVMTI_VTHREAD_STATE_NEW
+#define JVMTI_VTHREAD_STATE_NEW 0
+#endif
+	if (JVMTI_VTHREAD_STATE_NEW == vThreadState) {
+		vthreadCounts[0] += 1;
+	} else {
+		vthreadCounts[1] += 1;
+	}
+	return JVMTI_ITERATION_CONTINUE;
+}
+
 static jvmtiIterationControl
 continuationIteratorCallback(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData)
 {
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, object->object);
 	PORT_ACCESS_FROM_VMC(vmThread);
 	JavaCoreDumpWriter *jcw = (JavaCoreDumpWriter *)userData;
-	J9InternalVMFunctions *vmFuncs = vmThread->javaVM->internalVMFunctions;
 	j9object_t vthread = J9VMJDKINTERNALVMCONTINUATION_VTHREAD(vmThread, object->object);
-	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, object->object);
+	UDATA vThreadState = J9VMJAVALANGVIRTUALTHREAD_STATE(vmThread, vthread);
+	j9object_t threadObj = vthread;
+	BOOLEAN isMounted = FALSE;
```

Comment:
This should be `bool` to match the return type of `VM_ContinuationHelpers::isFullyMounted()`.